packages/react-common/src/moratorium-banner.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+export const CovidMoratoriumBanner: React.FC<{ locale?: string }> = ({
+  locale,
+}) => {
+  return locale === "es" ? (
+    <>
+      <b>Actualización COVID-19: </b>
+      JustFix.nyc está operativo, y hemos adaptado nuestros productos a las
+      normas establecidas durante la crisis de COVID-19. Todavía recomendamos
+      que se tomen todas las precauciones posibles para mantenerse sanos durante
+      esta crisis de salud pública. Gracias al poder de la organización de
+      inquilinos, los inquilinos no pueden ser desalojados por ninguna razón.
+      Visita las{" "}
+      <a
+        href="https://www.righttocounselnyc.org/ny_eviction_moratorium_faq"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <b>
```

Comment:
Err the extra `<b>` feels weird to me here, like it's redundant with the additional `<a>`.  It feels like styling rather than semantics, if that makes sense... is there any way we can remove it and keep the bold-ness of it reserved to CSS-land?

Reply:
Yep, that's fine. Honestly, we may not even need to bold that link at all— it's still pretty clear as it is with the default link styling on each of the respective products.